Tests/SwiftKueryPostgreSQLTests/TestSchema.swift

```diff
@@ -151,6 +155,7 @@ class TestSchema: XCTestCase {
                     }
                 }
             }
+            semaphore.wait()
```

Comment:
Can we not use expectations instead of semaphores in the tests? (I might be wrong, but I thought that was what they were for)